.github/workflows/iroha2-dev-nightly.yml

```diff
@@ -7,8 +7,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Login to DockerHub
-        uses: docker/login-action@v2
+      - uses: docker/setup-buildx-action@v2
```

Comment:
This is a bug. We **should have** buildx set up inside the container.

Reply:
I added `docker/setup-buildx-action` not because I saw errors but just because it seemed to be the best practice. I'll just remove it then